app/src/main/java/app/revanced/integrations/settingsmenu/SponsorBlockSettingsFragment.java

```diff
@@ -196,20 +160,43 @@ public void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    private void addGeneralCategory(final Context context, PreferenceScreen screen) {
-        final PreferenceCategory category = new PreferenceCategory(context);
+    private void addAppearanceCategory(Context context, PreferenceScreen screen) {
+        PreferenceCategory category = new PreferenceCategory(context);
         screen.addPreference(category);
-        category.setTitle(str("sb_general"));
+        category.setTitle(str("sb_appearance_category"));
+
+        votingEnabled = new SwitchPreference(context);
+        votingEnabled.setTitle(str("sb_enable_voting"));
+        votingEnabled.setSummaryOn(str("sb_enable_voting_sum_on"));
+        votingEnabled.setSummaryOff(str("sb_enable_voting_sum_off"));
+        category.addPreference(votingEnabled);
+        votingEnabled.setOnPreferenceChangeListener((preference1, newValue) -> {
+            SettingsEnum.SB_VOTING_ENABLED.saveValue(newValue);
+            updateUI();
+            return true;
+        });
```

Comment:
I've seen this code duplicate multiple times. It is possible to reduce duplicate code by creating a method with a generic type parameter constrained to Preference that takes in the resource names for the strings and the listener. But this can be something for another PR.